server/storage/endpoint/gc_key_space.go

```diff
@@ -51,7 +47,7 @@ type KeySpaceGCSafePointStorage interface {
 var _ KeySpaceGCSafePointStorage = (*StorageEndpoint)(nil)
 
 // SaveServiceSafePoint saves service safe point under given key-space.
-func (se *StorageEndpoint) SaveServiceSafePoint(spaceID string, ssp *ServiceSafePoint) error {
+func (se *StorageEndpoint) SaveServiceSafePoint(spaceID string, ssp *ServiceSafePoint, ttl int64) error {
```

Comment:
Could we store the ttl in the `ServiceSafePoint`, for example replace `ExpireAt`?